tooltester/src/main/java/io/dockstore/tooltester/TooltesterConfig.java

```diff
@@ -31,10 +32,11 @@ public TooltesterConfig() {
             setConfig(new HierarchicalINIConfiguration(configFile));
             setServerUrl(config.getString("server-url", "https://staging.dockstore.org/api"));
             setRunner(config.getString("runner", "cwltool cwl-runner cromwell").split(" "));
-            setDockstoreVersion(config.getString("dockstore-version", "1.6.0"));
+            setDockstoreVersion(config.getString("dockstore-version", "1.13.1"));
```

Comment:
Optionally, not to introduce too much feature creep, but you could get this from the CLI itself. That way there's no risk of a mismatch, and this code doesn't need to be updated.

Reply:
I have made it so the `server-url` and `token` are retrieved from `~/.dockstore/config` instead of `~/.tooltester/config`.